### PR TITLE
[Live Share: Integrated Chat] Remove vsls chat and contact provider

### PR DESCRIPTION
**Pull request by jramsay**
- Removes VSLS Chat and VSLS Contact Provider in preparation for Integrated Chat
- Rename 'Live Share Chat' to 'Chat'
- Remove VSLS specific viewlet chat injection
- Suppress some commands from showing up in the Command Palette that appear to be internal

@arjun27: Maybe I shouldn't be removing the VSLS Contact Provider? I've tested this works with the integrated chat but haven't verified Slack, Spaces (& Discord) are still working as expected. I was having trouble 'Signing in to Slack' as you mentioned the oAuth redirection isn't working. I wasn't quite sure how to get a Slack Token. Can I get your help with Slack & Spaces testing?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-<h1 align="center">Live Share Chat</h1>
+<h1 align="center">Chat</h1>
 
-<h3 align="center">Chat with your team while you collaborate over code using VS Live Share</h3>
```
Comment (jramsay) [inline README.md]: @lostintangent, @fubaduba: Can you fix up the readme?

File: package.json [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {
     "name": "chat",
-    "displayName": "Live Share Chat",
-    "description": "Chat with your team while you collaborate over code using VS Live Share",
-    "version": "0.31.0",
+    "displayName": "Chat",
```
Comment (jramsay) [inline package.json]: @lostintangent, @fubaduba: Also the displayName, description and version may need to be adjusted

File: package.json [hunk 3]
```diff
@@ -34,9 +34,6 @@
     "activationEvents": [
         "*"
     ],
-    "extensionDependencies": [
```
Comment (jramsay) [inline package.json]: @arjun27: I removed the Live Share extension dependency as I think it is no longer required. Maybe that is isn't correct though?

**Comment by arjun27** [comment]
Thanks for the PR! The code changes look good to me. I'll test the Slack and Spaces scenarios. I also want to update the readme to point users to the integrated chat experience and clarify the purpose of this extension.

File: README.md (hunk 1 above)
Comment (fubaduba) [reply README.md]: Yup.

**Comment by jramsay** [comment]
@arjun27: thanks!

**Comment by arjun27** [comment]
I've verified that things work properly, updated readme, and released a new version to the marketplace. Thanks for doing this @jramsay :)

**Pull request merged**